cli/dinghy/machine.rb

```diff
@@ -117,8 +118,16 @@ def system(*cmd)
     Kernel.system("docker-machine", *cmd)
   end
 
+  def preferences
```

Comment:
it'd be better to keep the preferences reading localized to only `cli.rb`, rather than starting to read them from other areas of the code as well... how about modifying the `DinghyCLI#machine` method to read the `machine_name` preference and modifying this class to take the machine name as an option